src/gui/canvas.cpp

```diff
@@ -71,7 +72,8 @@ canvas::canvas(QWidget* p)
     cm->register_command(new InteractiveDesAction<NEW>(m_working_set));   
     cm->register_command(new dicmdDesignSave(m_working_set));
     cm->register_command(new dicmdDesignLoad(m_working_set));
-        
+	cm->register_command(new InteractiveDeleteAction(m_working_set));
+	cm->register_command(new dicmdDeleteObj(m_working_set));
```

Comment:
Also add dicmdDeleteObj in command_manager.cpp registry so log can be replied ))

Reply:
Can't find dicmdCreateObj in command_manager.cpp